src/components/POLine/FundDistribution/FundDistributionForm.js

```diff
@@ -88,8 +82,7 @@ class FundDistributionForm extends Component {
             dataOptions={funds}
             fullWidth
             label={<FormattedMessage id="ui-orders.fundDistribution.id" />}
-            name={`${elem}.id`}
```

Comment:
`id` was removed and fix was applied in the Form and View components to support that